pkg/apis/wsecurity/v1/count_test.go

```diff
@@ -0,0 +1,30 @@
+package v1
+
+import (
+	"testing"
+)
+
+func TestCount_V1(t *testing.T) {
+	arguments := [][]uint8{
+		{4},
+		{5},
+		{1},
+		{254},
+		{255},
+		{0},
+	}
+	var args []interface{}
+	var profiles []ValueProfile
+	var piles []ValuePile
+	var configs []ValueConfig
+	for i := 0; i < 10; i++ {
+		profiles = append(profiles, new(CountProfile))
+		piles = append(piles, new(CountPile))
+		configs = append(configs, new(CountConfig))
+	}
+	for i := 0; i < len(arguments); i++ {
+		args = append(args, arguments[i])
+	}
+
+	ValueTests_All(t, profiles, piles, configs, args...)
```

Comment:
nit: we're only able to run the whole suite of tests this way... for example, `go test -run Basics ./...` will come back with "no tests to run". That's probably fine for now, but may be something we want to revisit down the line.

Reply:
In a way, the v1_tests are the basic tests, while we may need additional data type-specific tests to cover code path which is specific to the data-type. It is true that one of the benefits of going to an interface-based set of data types is that it reduces the need to write testing per data type.

Reply:
I'd be +1 on removing these tests until we've firmed up the interface (possibly by introducing the additional data types and accumulators mentioned in one of the above comments in this PR).

Reply:
I'd prefer to have a list of items to revisit since I am starting from a full repository and cutting slices in, adjusting all codes to the standards we mutually agree on along the way.